assets/blocks/course-outline/data.js

```diff
@@ -71,25 +71,34 @@ export const syncStructureToBlocks = ( structure, blocks ) => {
 	} );
 };
 
+const byStructure = ( target ) => ( { name, attributes } ) => {
+	const { id, type, title, lastTitle } = target;
+	if ( blockNames[ type ] !== name ) return false;
```

Comment:
I think we want to follow [WP Coding Standards for JavaScript](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/javascript/#blocks-and-curly-braces).